scripts/docker/buildpack-deps/Dockerfile.ubuntu2204.clang

```diff
@@ -33,8 +33,8 @@ RUN set -ex; \
 	apt-get update; \
 	apt-get install -qqy --no-install-recommends \
 		build-essential \
-		clang \
 		cmake \
+		clang-15 \
```

Comment:
What's the exact reason for the specific version meddling here, i.e. what exactly breaks without that? The very purpose of these images is to test something close to a default configuration, so messing with that too much is kinda against the point of these - and also strange that 20.04 gcc seems fine with whatever triggers this, while default 22.04 clang isn't :-).

Reply:
clang-15 is now the minimum required version to build evmone.

Reply:
> Before we install non-default versions in this one, we can also consider switching to 24.04 for the clang image.
> 
> We also need to add a gcc 24.04 image anyways by the way.

ossfuzz is still gonna be a problem since the Google team that maintains it doesn't really wanna upgrade their base images, which are still on focal (20.04) - https://github.com/google/oss-fuzz/issues/11835

All in all, kinda annoying :)

Reply:
Any problem with just pulling in the binaries for evmone?